public/index.html

```diff
@@ -20,6 +15,8 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="https://fonts.googleapis.com/css2?family=Fjalla+One&family=Merriweather+Sans:wght@300&display=swap"
+      rel="stylesheet">
```

Comment:
issue: Double-check the fonts you're pulling here with what's in Figma - I think the design requires may require more weights and/or styles. Check in with @ajkarczewski if you need help finding the page in Figma with the design elements on it.